### DISCUSSION TITLE
Rewrite tag with custom ones on api portal

### DISCUSSION THREAD

**Discussion by elmalakomar** [mia-platform/community#578] [Q&A] [answered]
### What product are you having troubles with?

Marketplace

### What Console version are you using?

v13.0.5

### Description

Hello,
I have an API portal that shows a lot of endpoints with tags assigned by the crud-service.

I would like to put a set of endpoints under the same tag.

I'm trying to follow this documentation: https://docs.mia-platform.eu/docs/runtime_suite/swagger-aggregator/configuration
and I wrote the services array using the transform paths and specifying the custom tags.

The problem is that, this configuration rewrites completely the swagger config, so I lose all the other exposed endpoints.

What I can do to reach my goal?

**Comment by LiliumCandidum** [comment, marked as answer]
Hi @elmalakomar, unfortunately it's not possible to "merge" your endpoints with the default swagger-aggregator endpoints so there is no solution for you problem at the moment
